lua/lspinstall/servers/lemminx.lua

```diff
@@ -0,0 +1,16 @@
+-- 1. get the default config from nvim-lspconfig
+local config = require"lspinstall/util".extract_config("lemminx")
+-- 2. update the cmd. relative paths are allowed, lspinstall automatically adjusts the cmd and cmd_cwd for us!
+config.default_config.cmd[1] = "./lemminx-0.18.0/mvnw"
```

Comment:
If I am correct, this shall be command line call to run the LSP server.

What I see is call to maven. When I tried to run it manually, it complained:
```
$ ./lemminx-0.18.0/mvnw 
./lemminx-0.18.0/mvnw: 219: cannot open /home/javl/install/lemmingx/ttt/lspinstall/.mvn/wrapper/maven-wrapper.properties: No such file
/home/javl/install/lemmingx/ttt/lspinstall/.mvn/wrapper/maven-wrapper.jar: No such file or directory
Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
```

When I run it from the directory:
```
$ cd lemminx-0.18.0/
$ ./mvnw 
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Eclipse LemMinX                                                    [pom]
[INFO] org.eclipse.lemminx                                                [jar]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Eclipse LemMinX 0.18.0:
[INFO] 
[INFO] Eclipse LemMinX .................................... SKIPPED
[INFO] org.eclipse.lemminx ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.086 s
[INFO] Finished at: 2021-10-01T23:12:53+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] No goals have been specified for this build. You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/NoGoalSpecifiedException
```

Reply:
I'm reading up more on the lspconfig, I thought `mvnw` would've been the executable lspinstall would've been looking for. That's my mistake.